ospfd/ospf_asbr.c

```diff
@@ -987,6 +987,7 @@ static void ospf_handle_external_aggr_update(struct ospf *ospf)
 				&aggr->match_extnl_hash,
```

Comment:
we are cleaning up hash as part of ospf_external_aggregator_free() api, so I think we need to remove this hash clean up call .

Reply:
This hash clean up call is using another free_func `ospf_aggr_handle_external_info` which performs additional things. Would it be appropriate to remove it?

Reply:
Yes it is required. But  here only concern is  hash_clean_and_free() api called twice . Anyhow, it is returning if hash is null. I think it should be fine.